### DISCUSSION TITLE
Purge exceptions by category?

### DISCUSSION THREAD

**Discussion by leinaht** [FreshRSS/FreshRSS#5348] [Q&A] [answered]
I'm not sure if this is an existing feature that I'm just missing, but I didn't find it in the documentation or by experimenting. Is there a way to exempt specific categories from the global purge rules? I'd like to never delete some feeds' entries (blogs, newsletters, Substack posts--already all in one "Newsletters" category) but use the regular purging rules on all other feeds. Thank you!

**Comment by math-GH** [comment, marked as answer]
Hi, yes there is an option. See the category configuration. There is an "Archiving" option to change from default to customized
